config/routes.rb

```diff
@@ -317,6 +317,7 @@ def show_step(name)
 
   resource :application_searches, path: 'application-searches', only: [:new] do
     post :search, on: :collection
+    get :search, on: :collection
```

Comment:
To avoid having personal information exposed in the query, we should avoid GET requests on search if possible.